### DISCUSSION TITLE
Some Zenstruck warning after start, preventing the server to start

### DISCUSSION THREAD

**Discussion by n0ntoxic** [benjaminjonard/koillection#1037] [Q&A] [answered]
Hi,
I just found your project and I like the superb documentation . However I stumbled across an error message that seems to rely on one of the used internal tools, I do not have any clue about. 

This is what the docker-compose log reads after "up":
```
koillection_1  | **** 11/11 - Setup complete, starting the server. ****
koillection_1  | [16-Jan-2024 16:28:45] NOTICE: PHP message: PHP Warning:  Can't preload unlinked class Zenstruck\Foundry\Factory@anonymous: Unknown parent Zenstruck\Foundry\Factory in /var/www/koillection/vendor/zenstruck/foundry/src/functions.php on line 45
^CGracefully stopping... (press Ctrl+C again to force)

```
This is where the overall process gets stuck.
I used the recommended Docker install method. Since I was already using Docker for other tools&services, this seemed like a no-brainer. The two files from the instructions have been created and adapted where needed.

Is there a dependancy I could have missed?

Thanks

**Comment by benjaminjonard** [comment, marked as answer]
Hi,
I believe everything is fine; PHP warnings shouldn't prevent the server from starting.
Maybe I should change the wording from "starting the server" to "server is running", because at this point it's normal that nothing else happens, the server is waiting for a request. 

Edit : 
I just checked my own instance, I have the same warning but everything works fine

**Reply by n0ntoxic** [reply]
Hi,
alright then it is indeed a different issue. Thanks for the clarification!
While waiting for any other log entries I have been trying to send requests from a browser in the same LAN, but also via curl from the server itself. But no connection at all. The server does not have an internal firewall, because there are several other LAN services running and I am using a lot of "custom" ports for these. Seems like the custom port I chose for koi was blocked though. When using the default 80 it works.
Thanks for ruling that out!
